src/NuGet.Core/NuGet.Protocol/Resources/MetadataResource.cs

```diff
@@ -64,4 +64,4 @@ public async Task<NuGetVersion> GetLatestVersion(string packageId, bool includeP
             return null;
         }
     }
-}
+}
```

Comment:
Revert please. There are no other changes.

Reply:
Fixed in https://github.com/NuGet/NuGet.Client/pull/2928/commits/70597649ad2c1ad5b63fc94e6b83aecde1812fae